src/Microsoft.OData.Client/Metadata/ClientTypeUtil.cs

```diff
@@ -17,7 +17,7 @@ namespace Microsoft.OData.Client.Metadata
     using System.Reflection;
     using Microsoft.OData.Metadata;
     using Microsoft.OData.Edm;
-    using c = Microsoft.OData.Client;
+    using C = Microsoft.OData.Client;

```

Comment:
In other places you use "Client"

Reply:
different idea in different day.  
😄 Let me change to use 'Client'.